include/ginkgo/core/base/device_matrix_data.hpp

```diff
@@ -175,14 +175,14 @@ class device_matrix_data {
      *
      * @return the number of stored elements of the matrix.
      */
-    size_type get_num_elems() const { return values_.get_num_elems(); }
```

Comment:
No deprecation of old function.
But I think this is a more tricky case, since this class already has a `get_size`, which returns something different then `size`.
IMO, we can't rename it here, and should just leave the old name.

Reply:
I wasn't aware we have another .get_num_elems member function. I'm looking for a better name, maybe match it with `Csr::get_num_stored_elements()`?

Reply:
I think that is a good approach.

Reply:
I think renaming it to `get_num_stored_elements` would still be appropriate.